superset-frontend/src/SqlLab/actions/sqlLab.test.js

```diff
@@ -24,22 +24,21 @@ import thunk from 'redux-thunk';
 import shortid from 'shortid';
 import * as featureFlags from 'src/featureFlags';
 import * as actions from 'src/SqlLab/actions/sqlLab';
-import { defaultQueryEditor, query } from '../fixtures';
+import { defaultQueryEditor, query, initialState } from 'src/SqlLab/fixtures';
 
 const middlewares = [thunk];
 const mockStore = configureMockStore(middlewares);
 
 describe('async actions', () => {
   const mockBigNumber = '9223372036854775807';
   const queryEditor = {
+    ...defaultQueryEditor,
     id: 'abcd',
     autorun: false,
-    dbId: null,
     latestQueryId: null,
-    selectedText: null,
     sql: 'SELECT *\nFROM\nWHERE',
     name: 'Untitled Query 1',
-    schemaOptions: [{ value: 'main', label: 'main', name: 'main' }],
+    schemaOptions: [{ value: 'main', label: 'main', title: 'main' }],
```

Comment:
why are we changing this to `title`

Reply:
@hughhhh This change(title -> name) accidentally happened [here](https://github.com/apache/superset/pull/20852/files#diff-c489b64153b9a09dc1f0984560850db74bf49af1184bfe68b9f258a44b89105aR42) while the PR made the change for queryEditor title to name.

cc: @lyndsiWilliams 

I reverted this change to pass the type role since I'm following [this type definition](https://github.com/apache/superset/blob/master/superset-frontend/src/SqlLab/components/SaveQuery/index.tsx#L50)

and it's also allocated as `title` here:

https://github.com/apache/superset/blob/eb5369f2a6f2dc238838119eb70194bf2b42b085/superset-frontend/src/components/DatabaseSelector/index.tsx#L224-L228